files/grest/rpc/address/address_info.sql

```diff
@@ -25,12 +26,16 @@ BEGIN
   SELECT
     SUM(tx_out.value)::text,
     SA.view,
+    bool_or(tx_out.address_has_script),
     COALESCE(
       JSON_AGG(
         JSON_BUILD_OBJECT(
           'tx_hash', ENCODE(tx.hash, 'hex'), 
           'tx_index', tx_out.index,
+          'block_height', block.block_no,
```

Comment:
Existing endpoints that reference block height are referenced as `height` only. Which one should we use to align across endpoints?

Reply:
yeah, height can mean more than block, like slot height.